moto/dynamodb/models/__init__.py

```diff
@@ -999,5 +1001,37 @@ def import_table(
     def describe_import(self, import_arn: str) -> TableImport:
         return self.table_imports[import_arn]
 
+    def export_table(
```

Comment:
Can we add a docstring to this method to explain what is implemented, and how it's different from AWS?

Per your note below, it doesn't matter if it doesn't exactly match the way AWS does things - as long as people are aware of it.